scripts/onboarding/templates/arc-k8s-extension/existingClusterOnboarding.json

```diff
@@ -13,14 +13,7 @@
       "metadata": {
         "description": "Location of the Azure Arc Connected Cluster Resource e.g. \"eastus\""
       }
-    },
-    "proxyEndpointUrl": {
-      "type": "string",
-      "defaultValue": "",
-      "metadata": {
-        "description": "If the cluster behind forward proxy, then specify Proxy Endpoint URL in this format: http(s)://<user>:<password>@<proxyhost>:<port>"
-      }
-    },

```

Comment:
Just want to confirm these settings in existingClusterOnboarding.json and existingClusterParam.json can be removed because if the proxy settings are needed, they _have_ to be settings for Arc K8s RP?

Reply:
> Just want to confirm these settings in existingClusterOnboarding.json and existingClusterParam.json can be removed because if the proxy settings are needed, they _have_ to be settings for Arc K8s RP?

> Yes, @gracewehner,  proxy settings are being populated by Arc K8s RP as part of our chart deployment. Before it was passed by user. Since this being passed by RP so this parameter not required in the template.